bin/tritond

```diff
@@ -0,0 +1,313 @@
+#!/bin/python
+
+# -*- coding: utf-8 -*-
+"""
+tritond
+~~~~~~~~
+
+Logging daemon for non-blocking triton events.
+
+Adapted from https://github.com/rhettg/BlueOx/blob/master/bin/oxd
+
+"""
+import argparse
+import errno
+import sys
+import logging
+import signal
+import struct
+import os
+import time
+import datetime
+from collections import defaultdict
+
+import zmq
+
+from triton import nonblocking_stream
+from triton.stream import get_stream
+from triton import config
+
+ENV_VAR_TRITON_CONFIG_PATH = 'TRITON_CONFIG_PATH'
```

Comment:
Every other triton tool uses `TRITON_CONFIG`

Probably some value in keeping that the same?